test/e2e/favicon.spec.js

```diff
@@ -8,22 +8,26 @@ test.describe('favicons', () => {
 		const responseManifest = await request.get(`${BASE_URL}/assets/manifest.json`);
 		const responsePng192 = await request.get(`${BASE_URL}/assets/icon_192x192.png`);
 		const responsePng512 = await request.get(`${BASE_URL}/assets/icon_512x512.png`);
+		const responsePng512_maskable = await request.get(`${BASE_URL}/assets/icon_512x512_maskable.png`);
 
 		expect(responseIco.ok()).toBeTruthy();
 		expect(responseManifest.ok).toBeTruthy();
 		expect(responsePng192.ok).toBeTruthy();
 		expect(responsePng512.ok).toBeTruthy();
+		expect(responsePng512_maskable.ok).toBeTruthy();
 		expect(await responseManifest.json()).toEqual({
 			icons: [
 				{
 					src: 'icon_512x512.png',
 					sizes: '512x512',
-					type: 'image/png'
+					type: 'image/png',
+					purpose: 'any'
 				},
 				{
-					src: 'icon_192x192.png',
-					sizes: '192x192',
-					type: 'image/png'
+					src: 'icon_512x512_maskable.png',
+					sizes: '512x512',
+					type: 'image/png',
+					purpose: 'maskable'
```

Comment:
Let's restore the test for the 192px image and additionally add the test for the maskable 512px image

Reply:
done